onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3146,9 +3152,8 @@ def aten_margin_ranking_loss(
 @torch_op("aten::masked_fill")
 def aten_masked_fill(self: TTensor, mask: BOOL, value: TTensor) -> TTensor:
     # masked_fill.Tensor(Tensor self, Tensor mask, Tensor value) -> Tensor
-    mask_cast = op.Cast(mask, to=BOOL.dtype)
     value_cast = op.CastLike(value, self)
```

Comment:
maybe add a note along this line?

```
# NOTE: Do not attempt to cast `mask` to BOOL because mask should not take any other types.
# `mask` coming in as other types is often an error and should fail the model.
```

Reply:
Done